src/commands/subcommands/sharedLabels.ts

```diff
@@ -0,0 +1,90 @@
+import { HttpStatusCode, IModify } from '@rocket.chat/apps-engine/definition/accessors';
+import { SlashCommandContext } from '@rocket.chat/apps-engine/definition/slashcommands';
+import { LayoutBlock } from '@rocket.chat/ui-kit';
+
+import { TodoistApp } from '../../../TodoistApp';
+import { MiscEnum } from '../../enums/Misc';
+import { getActionsBlock, getButton, getSectionBlock } from '../../helpers/blockBuilder';
+import { getSharedLabelsUrl } from '../../helpers/const';
+
+export async function sharedLabels(
+  app: TodoistApp,
+  modify: IModify,
+  context: SlashCommandContext
+): Promise<void> {
+  const user = context.getSender();
+  const room = context.getRoom();
+
+  const response = await app.getHttpHelperInstance().get(
```

Comment:
Let's create a SharedLabelsService.
That has a method called getSharedLabelsOmittingPersonalLabels, so this part will be a lot cleaner